Documentation/ExtensionArchitecture/Tutorials/Tea/DirectoryStructure.rst

```diff
@@ -0,0 +1,189 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Directory structure
+..  _extbase_tutorial_tea_extension_configuration:
+
+============================
+Create a directory structure
+============================
+
+Extbase requires or defaults to a certain directory structure. It is considered
+best practise to always stick to this structure.
+
+On the first level EXT:tea has the following structure:
+
+..  code-block:: none
+    :caption: Directory structure of EXT:tea
+
+    $ tree typo3conf/ext/tea
+    ├── Classes
+    ├── Configuration
+    ├── Documentation
+    ├── Resources
+    ├── Tests
+    ├── composer.json
+    ├── ext_emconf.php
+    ├── ...
+    └── README.md
+
+Directory :file:`Classes`
+-------------------------
+
+The folder :file:`Classes` has to contain all PHP classes provided by the
```

Comment:
Hmm. Maybe "should contain" instead of "has to contain" and further on "not available in the [default|recommended] autoloading". In can also use "src" folder, but have to adjust the autoloading ;-)
Or "has to contain all PHP classes by convention".
Just as a food for thought ;-)